### PR TITLE
Shields fix

### PR DESCRIPTION
**Pull request by soslow4u**
## About The Pull Request

- Moved conduits spawn to main generator
- Removing single conduit would make generator remove whole batch
- Deploying conduits allows you to setup hull and multi z shields

New shield generators won't make any messages still

## Changelog
:cl:
fix: You can build real shield generators from boards. Right-click and deploy conduits for multi dock and hull shielding options
/:cl:

### CODE REVIEW

**Comment by GrayRachnid** [comment]
hull.dm deploys conduits when it is built, the normal one does not. Conduits are currently a snowflake structure, the order asked to make them fixable/removeable.

This is up to the maintainers to merge but a sloppy fix like this should not be accepted in my opinion.

**Comment by soslow4u** [comment]
Such slowflakey item is hard-linked to hull generator and they pops in when *hull* core was unanchored from floor.
So you want to keep those, right. Let me pour a bit more code there then

**Comment by soslow4u** [comment]
updated with conduits handling and such stuff

**Comment by soslow4u** [comment]
fine

File: code/modules/shield_generators/hull.dm [hunk 1]
```diff
@@ -2,133 +2,9 @@
...
-			toggle_tendrils(FALSE)
-			anchored = FALSE
-		else
-			if(istype(get_turf(src), /turf/space)) return //No wrenching these in space!
-			if ((toggle_tendrils(TRUE)))
-				to_chat(user, SPAN_NOTICE("You secure the [src] to the floor!"))
-				anchored = TRUE
-		return
\ No newline at end of file
+	current_energy = max_energy * 0.30
```
Comment (TheShown911) [inline code/modules/shield_generators/hull.dm:10]: No newline.

**Review by TheShown911 — COMMENTED** [review]
Looks good

**Review by TorinoFermic — APPROVED** [review]

**Pull request merged**